crates/ef-testing/src/evm_sequencer/evm_state.rs

```diff
@@ -191,6 +197,50 @@ impl EvmState for KakarotSequencer {
             Into::<FieldElement>::into(nonce).to_bytes_be(),
         ))
     }
+
+    fn get_code_at(&mut self, evm_address: &Address) -> StateResult<Bytes> {
+        let starknet_address = compute_starknet_address(evm_address);
+
+        let bytecode_len = (&mut self.0.state).get_storage_at(
+            starknet_address.try_into()?,
+            get_storage_var_address("bytecode_len_", &[])?,
+        )?;
+        let bytecode_len: u64 = bytecode_len.try_into()?;
+        if bytecode_len == 0 {
+            return Ok(Bytes::default());
+        }
+
+        let full_16 = bytecode_len / 16;
```

Comment:
slightly confusing name, can you find something else?


can we also add comments?

-> we're assuming here that we pack 16 bytes of bytecode in the CA (which is ok, just need to document it imo, it'll change for ssj)